### PR TITLE
docs: adding deployment instructions 

### LINKED ISSUE #11: docs: Document deployment process for Netlify and Partykit.
Based off of #10, we need to document the deployment process.

Setting up environment variables:

1. Run `npx partykit@latest token generate` to generate a token.
2. Run `ntl env:set PARTYKIT_LOGIN nickytonline` to add it to the environment variables for Netlify deploys.
3. Run `ntl env:set PARTYKIT_TOKEN THE_GENERATED_SUPER_SECRET_TOKEN` to add it to the environment variables for Netlify deploys.
4. Run `ntl env:set PUBLIC_PARTYKIT_HOST https://astro-partykit-starter-party.nickytonline.partykit.dev` so that the production URL for Partykit is available for the deployment.

Deployment to Netliffy:

Point to Netlify deployment docs (maybe this one https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) and also using the Netlify CLI for deployments.

Partykit deployment:

This is handled once #10 is closed, but it'd be good to mention the Partykit deployment docs at https://docs.partykit.io/guides/deploying-your-partykit-server.

### PR DESCRIPTION
**Pull request by jdwilkin4**
## Description

This PR adds the instructions for setting up env variables, and updates the sections for deploying to Netlify and Partykit.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] 📦 Chore
- [ ] ⏩ Revert

## Related Tickets & Documents
closes #11 

## Added to documentation?

- [x] 📜 README.md
- [ ] 📜 CONTRIBUTING.md
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,13 @@ Any static assets, like images, can be placed in the `public/` directory.
 
 ## Deployment
 
+### Setting up Environment Variables
+
+1. Run `npx partykit@latest token generate` to generate a token.
+2. Run `ntl env:set PARTYKIT_LOGIN choose-your-value-here` to add it to the environment variables for Netlify deploys.
```
Comment (jdwilkin4) [inline README.md:59]: I wasn't sure what to use as a placeholder here for the value.

The way I understood the [docs](https://cli.netlify.com/commands/env#envset), was that `env:set` command accepts a key value pair.
In the open issue, the placeholder value was set to `nickytonline` but I am assuming people using this for their own personal projects will want to use their own username.

So maybe this should say `ntl env:set PARTYKIT_LOGIN add-username-here`?

Comment (nickytonline) [reply README.md:59]: > So maybe this should say ntl env:set PARTYKIT_LOGIN add-username-here?

I'd go with `ntl env:set PARTYKIT_LOGIN your-username`

Comment (nickytonline) [reply README.md:59]: The template is meant to deploy to Netlify. They can obviously deploy elsewhere, so I wonder if we should explicitly mention that `ntl env:set` is for Netlify only and if you deploy elsewhere to refer to their docs on setting environment variables for an application?

Comment (jdwilkin4) [reply README.md:59]: > so I wonder if we should explicitly mention that ntl env:set is for Netlify only and if you deploy elsewhere to refer to their docs on setting environment variables for an application?

Yeah, I think it is fine to add a note like that

**Review by adiati98 — APPROVED** [review]

**Review by nickytonline — APPROVED** [review]
Thanks @jdwilkin4! :shipit:

**Pull request merged**